docs/en/serverless/slos/create-an-slo.mdx

```diff
@@ -161,6 +161,27 @@ When defining either an APM latency or APM availability SLI, set the following f
 * **Threshold (APM latency only):** The latency threshold in milliseconds (ms) to consider the request as good.
 * **Query filter:** An optional query filter on the APM data.
 
+<div id="synthetics-availability-sli"></div>
```

Comment:
Can you remind me what this is for again? Is it for linking?

Reply:
Yes!